packages/mobile/src/components/audio/GoogleCast.tsx

```diff
@@ -73,7 +73,7 @@ export const useChromecast = () => {
   // Load media when the cast connects
   useEffect(() => {
     if (castState === CastState.CONNECTED) {
-      loadCast(track)
+      loadCast(track, global.progress.currentTime ?? 0)
```

Comment:
I think this make sense. In the past, we've had bugs where we assumed 0 time when it wasn't correct, but i think this is right